### ISSUE TITLE
problems building several bioconda recipes (ete3 optional dependencies)

### ISSUE THREAD

**Issue by jhcepas** [jhcepas/bioconda-recipes#1]
@bgruening, @nsoranzo , any idea why this is still failing on Travis?: https://travis-ci.org/jhcepas/bioconda-recipes/builds/189407297
Everything compiles fine locally on my macbook (also with --docker). 

- The linux build runs ok on Travis, except for a weird "bad interpreter" error when compiling phyml.
- The OSX build fails completely, but don't know why. The same recipes seem to compile well when tested individually (https://travis-ci.org/jhcepas/bioconda-recipes/builds/189406502) 

Drafts recipes under testing: https://github.com/bioconda/bioconda-recipes/compare/master...jhcepas:phylotools

**Comment by bgruening** [comment]
@jhcepas can you add this to your build.sh please: https://github.com/bioconda/bioconda-recipes/blob/27ebbe045174267682315e008b6e682f416b7ca8/recipes/snp-sites/build.sh#L2-L12

**Comment by nsoranzo** [comment]
Yes, automake has been fixed in conda-forge, but you may need to require >=1.15 in meta.yaml . autoconf is not fixed yet.

**Comment by jhcepas** [comment]
thanks! that solved the problem with phyml on linux. 
The OSX builds still fail, but work locally. Any clue?

**Comment by bgruening** [comment]
Do you have a link? I doubt I can help fixing OSX errors, but I can look at it.
You can also skip a build, then no OSX package will be build.

**Comment by jhcepas** [comment]
Here is the travis log. https://travis-ci.org/jhcepas/bioconda-recipes/jobs/189480870
Cannot see much there... (except for the "find" error)

**Comment by jhcepas** [comment]
Check, for instance, paml compilation. It fails in the previous OSX build linked, but worked if the same recipe is pushed independently in a different branch: https://travis-ci.org/jhcepas/bioconda-recipes/jobs/189406504

**Comment by bgruening** [comment]
Mh, https://travis-ci.org/jhcepas/bioconda-recipes/jobs/189480870#L349
A file could not be found ...

@jhcepas if you have submit this separate branch as PR and we will see what happens. Thanks for working so long on your branch, now it's time to go public :)

**Comment by jhcepas** [comment]
Oki, done :)
